### ISSUE TITLE
Nameservers array merges on run

### ISSUE THREAD

**Issue by jonathantullett** [chr4-cookbooks/resolvconf#17]
With the default['resolvconf']['nameserver'] settings being an array in attributes/default.rb, when you try and override them in the node config
```
  "default_attributes": {
        "resolvconf": {
                "nameserver": ["192.168.22.24"],
        }
  },

```
The end result is:
```
nameserver 208.67.222.222
nameserver 208.67.220.220
nameserver 192.168.22.24
```

This is because on run, arrays are merged.  What's the best way around this using your cookbook? Maintaining a custom version isn't ideal.

Thanks!

**Comment by jonathantullett** [comment]
For now, I have edited your default attributes and removed the default nameserver entries and it's working fine. Can this be considered for the official release?

**Comment by chr4** [comment]
Released in v1.0.0
